security/kaspersky_server.ksy

```diff
@@ -0,0 +1,13 @@
+meta:
+  id: kasperskyserver
+  endian: le
+  title: Kaspersky for Windows Server quarantine file parser
+  license: CC-BY-SA-4.0
+  ks-version: 0.9
+doc: |
+  Creator: Florian Bausch, ERNW Research GmbH, https://ernw-research.de
+  License: CC-BY-SA-4.0 https://creativecommons.org/licenses/by-sa/4.0/
+seq:
+  - id: mal_file
```

Comment:
What's inside?

Reply:
It's the malicious file, quarantined by Kaspersky for Windows Server

Reply:
Thank you. Can it be documented in the spec?